submissions/damilolaSalami/domManipulation/signUpform.js

```diff
@@ -0,0 +1,57 @@
+const submitform = document.getElementById("validateForm"),
+    firstname = document.getElementById("firstname"),
+    lastname = document.getElementById("lastname"),
+    phonenumber = document.getElementById("phoneNumber"),
+    email = document.getElementById("email"),
+    password = document.getElementById("password"),
+    confirmpassword= document.getElementById("confirmpassword");
+
+const nameval =  /^[a-zA-Z ]+$/,
+    pnumberval = /^[+]*[0-9]+$/,
+    emailval = /^\w+([-]?\w+)*@\w+([-]?\w+)*(\.\w{2,6})+$/,
+    passwordval = /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[._$,/;:+=~`'"<>^#?!@$%^&*-]).{6,}$/;
+
+function formValidation (field, Regex, error_one, error_two, event) {
+    if (field.value === "") {
+        field.nextElementSibling.innerHTML = error_one;
+        event.preventDefault();
+        return false;
+    } else if (!field.value.match(Regex)) {
+        field.nextElementSibling.innerHTML = error_two;
+        event.preventDefault();
+        return false;
+    } else {
+        field.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function pword(event) {
+    pasword = password.value;
+    cpasword = confirmpassword.value;
+    if (cpasword == "") {
```

Comment:
No need to check if it is empty. We are already handling that in `formValidation` function